java/com/google/flatbuffers/Table.java

```diff
@@ -56,6 +56,23 @@ protected Charset initialValue() {
    */
   public ByteBuffer getByteBuffer() { return bb; }
 
+  public <T> T getScalarField(int field, T defaultValue, Class<T> type) {
+    int offset = __offset(field);
```

Comment:
Stupid things. I was sure it was something existing in the cpp version but now as I'm checking it... It is not the case. Therefore I remove it.